stm32-modules/heater-shaker/firmware/host_comms_task/usbd_conf.c

```diff
@@ -64,7 +64,7 @@ void HAL_PCD_MspInit(PCD_HandleTypeDef *hpcd)
   __HAL_RCC_SYSCFG_CLK_ENABLE();
 
   /* Set USB Default FS Interrupt priority */
-  HAL_NVIC_SetPriority(USB_LP_CAN_RX0_IRQn, 5, 0);
+  HAL_NVIC_SetPriority(USB_LP_CAN_RX0_IRQn, 10, 0);
```

Comment:
I thought you said in the description that the USB was not actually causing issues. Do we need to change the priority of this interrupt then?

Reply:
well, we _probably_ don't. but in general it's nice to keep things deprioritized to avoid running out of priority space.